### ISSUE TITLE
"Copy property" feature in Blender Bim addon is not re-implemented in latest versions

### ISSUE THREAD

**Issue by avicomartin** [IfcOpenShell/IfcOpenShell#1619]
In previous versions, you can select a group of object and assign user pset at time to all them. Now, pset are assigned only to the first object in the selection.
The same occurs trying to copy properties pset for one objects to the rest of selection. Now you need make editable every object you can fill the property and copy the value using "copy to selected".
Old behavior was very interesting because if you add many objects to your project from other source (cad translation for example) and need assign psets and properties to all of them, do-in in a "massive way" is useful but add pset and make it "editable edit" for every object is not a viable workflow if you have a lot of objects (testing a "work case" in my work, I need assign at least four psets to every object and every pset has a lot of properties, some of them common to all objects. A group can arrive to 350 units so make a "individual assignment" is not a viable workflow)

**Comment by fontant** [comment]
hi, how can it work? can you help me to understand how to Upgrade BlenderBIM with these new operators?
Thanks

**Comment by Moult** [comment]
You can download the latest from https://blenderbim.org/

**Comment by avicomartin** [comment]
Add properties to an object and save. Later, select objects you want add this property and in last position the object with properties. Then edit to change properties and pick the copy button. This replicate the property to all objects selected. Selection order is important; you must select last the object with properties you want copy

**Comment by fontant** [comment]
No chance to get it without your help!
But unluckily it doesn't want work, this is the error 
![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/109509908/57c13cb8-f8b4-4453-9f26-5fd0a5f1bf47)
I already resinstalled Blender and BlenderBIM

**Comment by Moult** [comment]
If you hop by https://osarch.org/chat around the 9-5pm sydney timezone we can screenshare and help you debug.

**Comment by fontant** [comment]
We have a completely reverse fusorarium, I'm in italy but I'll try anyway.
But I had a doubt, I am trying to copy values from a custom pset, maybe blender bim can't handle it?

**Comment by Moult** [comment]
You're right, specifically for custom psets, it didn't work. This is now fixed.

**Comment by fontant** [comment]
Thanks so much!
